### ISSUE TITLE
Translations: NWS abbreviations

### ISSUE THREAD

**Issue by colinmurphy01** [weather-gov/weather.gov#1625]

**Comment by colinmurphy01** [comment]
Will do for Chinese, not for spanish.

**Comment by sspj-does-weather** [comment]
Update on this: The site and social media metadata don't use the same twig translation filters as other content. I've created #2168 to spend more time on the issue. The occurrence on the landing page is in the middle of block of text which needs to be marked for translation (and then translated). I've created #2166 for this work. The header and footer are the easiest, but they also aren't marked for translation yet, so I think it would be reasonable to do #2166 first. I've copied the strings from this ticket to [the spreadsheet](https://docs.google.com/spreadsheets/d/1xsCSnnQ9m4BIHCGbP6GGgMz95VKHQQvOvkbedyaLLm8/edit) 🔒 I've got going in the Translations folder so they won't be lost.

tl;dr: My sizing estimation was wrong, because I thought this was about copying & pasting a few strings. It turns out that more work is needed, so I've split this into sensible chunks of work for your prioritization.

@mel-keene-gsa and @colinmurphy01 is there anything else you'd like to see here or is this good to close?
